src/ui/popup/site-list-page/index.tsx

```diff
@@ -25,10 +25,10 @@ export default function SiteListPage(props: ViewProps) {
             <CheckButton
                 checked={props.data.settings.enableForProtectedPages}
                 onChange={onEnableForProtectedPages}
-                label={'Enable on protected pages'}
+                label={'Disable on protected pages'}
```

Comment:
https://github.com/darkreader/darkreader/pull/3426#discussion_r474659318

Reply:
OK, but that will take much longer to craft. Flipping the sign of preference was the quickest fix making it back to working order.

Reply:
> OK, but that will take much longer to craft. 

Took maximal 10 minutes to debug it and fix it.